src/_locales/pt-BR/messages.json

```diff
@@ -0,0 +1,601 @@
+  },
+  "optionSearchBoxIntegrationDescr": {
+    "message": "Permite que você pesquise emojis na barra de endereço, iniciando sua pesquisa com “emoji”, seguido por um espaço.",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "7c572ec9de6937cf1c57d9c8d71384d8"
+  },
+  "optionSearchCopyAction": {
+    "message": "Copiar para área de transferência",
+    "description": "This is an radio option shown in the add-on settings.",
+    "hash": "17b6a865f4e197dcff670c9ac12517a1"
+  },
+  "optionEmojipediaAction": {
+    "message": "Buscar em Emojipedia",
+    "description": "This is an radio option shown in the add-on settings.",
+    "hash": "29ee502aa06150b1345b1a80bbcd0df2"
+  },
+  "optionImageAltTextSearchBarDemo": {
+    "message": "Exemplo que mostra uma pesquisa na barra de endereço com o emoji para o termo “cachorro”, para o qual são sugeridos vários emojis.",
+    "description": "This is the alt description for an image shown in the add-on settings.",
+    "hash": "4cb29d49481d62a696ee8698d490ec58"
+  },
+  "optionsLicenseDisclaimer": {
+    "message": "Todas as imagens de emoji podem estar sujeitas a licenças diferentes. Para obter mais informações, consulte $LINK$.",
+    "description": "This is a text shown in the add-on settings explaining the license terms.",
+    "placeholders": {
+      "link": { "content": "$1", "example": "<a href=\"https://github.com/rugk/awesome-emoji-picker/blob/master/LICENSE.md\">the whole license file</a>" }
+    },
+    "hash": "18dfd0f38e1d38504273202e88121e67"
+  },
+  "optionsLicenseDisclaimerLink": {
+    "message": "https://github.com/rugk/awesome-emoji-picker/blob/master/LICENSE.md",
+    "description": "This is the link to the LICENSE file of this add-on. Used in the license disclaimer, see optionsLicenseDisclaimer. Usually, you do not need to translate this as we have only an English version.",
+    "hash": "a941d363fa723fc089f7d9600e292c1f"
+  },
+  "optionsLicenseDisclaimerLinkText": {
+    "message": "o arquivo inteiro da licença",
+    "description": "This is text of the link to the LICENSE file of this add-on. Used in the license disclaimer, see optionsLicenseDisclaimer.",
+    "hash": "3471458865aafdd0f3335877239d5d91"
+  },
+  "translatorCredit": {
+    "message": "Esta extensão foi traduzida do inglês por $TRANSLATORS$.",
+    "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+    "placeholders": {
+      "translators": { "content": "$1", "example": "<a href=\"https://github.com/rugk/\">@rugk</a>" }
+    },
+    "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
+  },
+  "translatorLink": {
+    "message": "https://github.com/rugk",
+    "description": "The link to the translator's GitHub profile.",
+    "hash": "4775fb2f88fd33d02b871e5021dec3e9"
+  },
+  "translatorUsername": {
+    "message": "rugk",
```

Comment:
I guess you should add yourself here (and link to you above). See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on